ocp_resources/resource.py

```diff
@@ -15,7 +15,6 @@
 import kubernetes
 from kubernetes.dynamic import DynamicClient, ResourceInstance
 import yaml
-from benedict import benedict
```

Comment:
`benedict` is no longer used and should be removed from the project.

Reply:
ok. Will do.